formfactory/models.py

```diff
@@ -133,9 +133,6 @@ class Meta(object):
     def __unicode__(self):
         return self.title
 
-    def absolute_url(self):
```

Comment:
I do understand that absolute_url makes for a nice property but the issue is the FED guys already use get_absolute_url in the other apps. This introduces ambiguity.